src/app/icd/server/tests/ICDConfigurationDataTestAccess.h

```diff
@@ -39,6 +39,8 @@ class ICDConfigurationDataTestAccess
         return mData->SetModeDurations(activeModeDuration, idleModeDuration);
     }
 
+    void SetFeatureMap(uint32_t featureMap) { mData->SetFeatureMap(featureMap); }
```

Comment:
Why is this taking an integer, instead of the right bitmask?  Then we could be type-safe and as a bonus the tests would be self-documenting instead of needing comments explaining what magic numbers like 0x07 mean.

Reply:
Changed it to use `BitFlags<app::Clusters::IcdManagement::Feature>`